kcp/pkg/util/watcher_util.go

```diff
@@ -282,20 +295,24 @@ func PerformIstioGWCheck(ctx context.Context, istioClientSet *istioclient.Client
 }
 
 func PerformIstioVirtualServiceCheck(ctx context.Context, istioClientSet *istioclient.Clientset,
```

Comment:
Could you remove this function and unify logic with HandleProcessing logic from the reconciler. They both do the same thing!

Reply:
I removed this function as Istio Gateway configuration will not change.

Reply:
I will unify this when we refactor to use the `watch` mechanism [here](https://github.com/kyma-project/runtime-watcher/issues/67).